File: design-patterns/java/java-observer-design-pattern/Post.java
public class Post {
    private Followed followed;
    private String content;

    public Post(Followed followed, String content) {
        this.followed = followed;
        this.content = content;
    }

    public Followed getFollowed() {
        return followed;
    }

    public void setFollowed(Followed followed) {
        this.followed = followed;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    @Override
    public String toString() {
        return this.followed.getName();
    }
}

File: design-patterns/csharp/csharp-observer-design-pattern/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_observer_design_pattern
{
    public class Post
    {
        private Followed followed;
        private string content;

        public Post(Followed followed, string content)
        {
            this.followed = followed;
            this.content = content;
        }

        public Followed Followed
        {
            get { return followed; }
            set { followed = value; }
        }

        public string Content
        {
            get { return content; }
            set { content = value; }
        }

        public override string ToString()
        {
            return this.followed.Name;
        }
    }
}